source/kraken/worker.h

```diff
@@ -59,13 +59,15 @@ struct JourneysArg {
     type::RTLevel rt_level;
     std::vector<type::EntryPoint> destinations;
     std::vector<uint64_t> datetimes;
+    type::EntryPoint isochrone_center;
```

Comment:
i'd make this optional isochrone_center, if i were you

Reply:
Yes, I hesitated between this version, a `boost::optionnal` or a `unique_ptr`